packages/perseus/src/widgets/input-number/input-number-validator.test.ts

```diff
@@ -0,0 +1,145 @@
+import {testDependencies} from "../../../../../testing/test-dependencies";
+import * as Dependencies from "../../dependencies";
+import {mockStrings} from "../../strings";
+
+import inputNumberValidator from "./input-number-validator";
+
+import type {Rubric} from "./input-number.types";
+
+describe("static function validate", () => {
```

Comment:
```suggestion
describe("inputNumberValidator", () => {
```
It's not a static method in this context.